lib/new_relic/agent/llm/llm_event.rb

```diff
@@ -0,0 +1,41 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+module NewRelic
+  module Agent
+    module Llm
+      class LlmEvent
```

Comment:
I don't know if this is the best name for this class. `Default` or `Event` also come to mind. However, I'm not opposed to keeping `LlmEvent` as-is.

Reply:
I don't mind `LlmEvent`, but I don't know if thats just because I've seen it a lot. I like that in a way, the longer name seems more 'grand' and as the parent of all the other events I notice it more.

Reply:
Thanks! Let's leave it for now!